empress/__init__.py

```diff
@@ -140,6 +140,19 @@ def draw_stats(self):
         self.draw_stats_on(ax)
         return figure
 
+    def stats(self, num_trials: int = 50):
```

Comment:
Duplicate of line 130.

Reply:
Please change this default value to 100.  Thanks!

Reply:
I am not sure what this is. Should I make the change?

Reply:
Removed.